scripts/newinstall.sh

```diff
@@ -638,16 +638,17 @@ n8l::python_check() {
 			cat <<-EOF
 
 			LSST stack requires Python 2 (>=2.${minver2}) or 3 (>=3.${minver3}); you
-			seem to have $($py_interp -V 2>&1) on your path ($(which $py_interp)).
+			seem to have $($py_interp -V 2>&1) on your path ($(command -v $py_interp))
+			.
```

Comment:
Why is this period on a line by itself?

Reply:
Otherwise the preceding line would be 81 cols wide (with a tabstop spacing of 2).  We need to figure out a reliable way of line wrapping output so the heredocs don't have be verbatim formatted.

Reply:
when the message prints out what does it look like then? Is the period on a line by itself or does it all look fine?

Reply:
It ends up on a line by itself.

Reply:
Which is worse than it being in column 81. I'd rather remove it completely or move the text after `your path` to the new line to keep below 81.

Reply:
It looks like both `fold` and `fmt` are builtins -- `fold` doesn't seem to back fill so I've attempted to use `fmt` to format the output to 78 cols.

Reply:
(note the correction above -- I originally said "builtins" when I meant part of `coreutils`.